src/api/app/controllers/source_controller.rb

```diff
@@ -110,25 +102,12 @@ def show_project
       return
     end
 
-    render xml: render_project_packages
+    render_project_packages
   end
 
   def render_project_packages
-    packages = nil
-    if params.key? :expand
-      packages = @project.expand_all_packages
-    else
-      packages = @project.packages.pluck(:name)
-    end
-    output = ''
-    output << "<directory count='#{packages.length}'>\n"
-    if params.key? :expand
-      output << packages.map { |p| "  <entry name=\"#{p[0]}\" originproject=\"#{p[1]}\"/>\n" }.join
-    else
-      output << packages.map { |p| "  <entry name=\"#{p}\"/>\n" }.join
-    end
-    output << "</directory>\n"
-    output
+    @packages = params.key?(:expand) ? @project.expand_all_packages : @project.packages.pluck(:name)
```

Comment:
Same here

Reply:
here I think package_names doesnt apply when the parameter ```:expand``` is given..

Reply:
Ah, I see. then keep it that way.